### DISCUSSION TITLE
Scope Issue for admiral 0.12.0

### DISCUSSION THREAD

**Discussion by bms63** [pharmaverse/admiral#1970] [General]
Hi all,

@pharmaverse/admiral 

Rather than making an issue, I thought we would try out the Discussion ability to help discuss the Scope for admiral 0.12.0 release.  My goal is to really help keep the Issue Backlog cleaner in upcoming and future releases.  Hopefully, if this is successful we can move other "Discussion" Issues out of the backlog and over to here.  

What say you admiral team!?! What is our drive and focus for this release.  I put in a few, please respond and I will update the list! 

Definitely Doing:
- [ ] DAIDs Grading Criteria
- [x] Retire `admiral.test` in favor of `admiraldata`
- [x] Prioritize deprecating and aligning functions needed to bring admiral to 1.0 status
- [ ] Review of TA packages for code that could be adopted in core admiral
- [ ] 

Probably should do:
 - [ ] Move Discussion Issues to actual Discussion Section to keep cleaner Issue Backlog

Dreaming
- [ ] Roclet updates and increase findability of documentation

**Comment by manciniedoardo** [comment]
One thing we probably should do is evaluate the contents of the TA extension packages to see if anything in there is TA-agnostic enough to be worth integrating in `{admiral}`. 

This suggestion is borne out of general discussions around the scope of the TA extension packages. In summary, as `{admiral}` grows and expands its capabilities it is natural that _some_ of the functionality previously relegated to the extension packages can be addressed with core functions. Then, extension packages would only retain what is _truly_ specific to that TA as well as useful vignettes/templates.

**Comment by bms63** [comment]
As we at `0.12` going to close the scope discussion
